core/src/test/java/org/springframework/security/scheduling/DelegatingSecurityContextTaskSchedulerTests.java

```diff
@@ -58,7 +61,8 @@
 	@Before
 	public void setup() {
 		MockitoAnnotations.initMocks(this);
-		this.delegatingSecurityContextTaskScheduler = new DelegatingSecurityContextTaskScheduler(this.scheduler);
+		this.delegatingSecurityContextTaskScheduler = new DelegatingSecurityContextTaskScheduler(this.scheduler,
```

Comment:
Should there be a test that confirms that the security context was indeed used?

Reply:
I've added two tests in d455211, for explicit/current security context